org.eclipse.scanning.test/src/org/eclipse/scanning/test/command/ScanRequestWithProcessingTest.java

```diff
@@ -54,7 +54,7 @@ public static void disconnect()  throws Exception {
 	@Test
 	public void testGridScanWithProcessing() throws Exception {
 
-		String cmd = "sr = scan_request(grid(axes=('yNex', 'xNex'), start=(0, 0), stop=(3, 3), count=(2, 2), snake=False), "
+		String cmd = "sr = scan_request(grid(axes=('yNex', 'xNex'), start=(0, 0), stop=(3, 3), count=(2, 2), snake=False, continuous=False), "
                 + "det=[detector('mandelbrot', 0.1), detector('processing', -1, detectorName='mandelbrot', processingFilePath='/tmp/sum.nxs')],"
                 + "file='"+path+"' )";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "file='" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)